apps/content/api/portal/access_requests.py

```
@@ -0,0 +1,180 @@
+
+
+def _get_service() -> AssetAccessRequestService:
+    return AssetAccessRequestService(AssetAccessRequestRepository())
+
+
+@router.get("access-requests/", response=list[AccessRequestListOut])
+@permission_required([permission_class(PermissionChoice.PORTAL_VIEW_ACCESS_REQUESTS)])
+@paginate
+@ordering(ordering_fields=["created_at", "id"])
+@searching(search_fields=["developer_user__name", "developer_user__email", "asset__name"])
+def list_access_requests(request: Request, filters: AccessRequestFilter = Query(...)):
```

Ruff (0.15.17)

[warning] Do not perform function call `Query` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)